pkg/syncer/cnsoperator/manager/init.go

```diff
@@ -181,8 +190,8 @@ func InitWcpCnsOperator(configInfo *types.ConfigInfo, coInitParams *interface{})
 	return nil
 }
 
-// InitCommonCnsOperator initializes the Cns Operator for all flavors
-func InitCommonCnsOperator(ctx context.Context, clusterFlavor cnstypes.CnsClusterFlavor,
+// InitCommonModules initializes the common modules for all flavors
+func InitCommonModules(ctx context.Context, clusterFlavor cnstypes.CnsClusterFlavor,
```

Comment:
The method name & file name does not match. Can we move this method to a new file like: `syncer/commonmodules `or something similar

Reply:
We don't anticipate anything other than this function in `syncer/commonmodules` if we have it.
However I feel its OK to have it here as there are common modules that are initialized for the cns operator functionality only.